.github/workflows/doc-automation.yml

```diff
@@ -5,29 +5,23 @@ on:
 jobs:
   build_docs_job:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.8]
     steps:
+    - name: Setup Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+        architecture: x64
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Dependencies
+    - name: Install Dependencies
       run: |
-        echo `python3 --version`
-        sudo apt-get install -y python-setuptools
-        python3 -m pip install --upgrade pip
-        python3 -m pip install -U sphinx
-        python3 -m pip install setuptools
-        python3 -m pip install torch
         git clone https://github.com/pytorch/pytorch_sphinx_theme
-        python3 -m pip install -e pytorch_sphinx_theme
-        python3 -m pip install recommonmark
-        python3 -m pip install katex
-        python3 -m pip install sphinxcontrib.katex
-        python3 -m pip install sphinx_markdown_tables
-        python3 -m pip install torchserve torch-model-archiver torch-workflow-archiver
+        pip install -e pytorch_sphinx_theme
```

Comment:
do you need to do it? I think you are installing the theme in requirements.txt, no?